app/frontend/src/features/auth/useAuthStore.test.ts

```diff
@@ -39,9 +39,9 @@ describe("usePersistedState hook", () => {
 describe("useAuthStore hook", () => {
   it("sets and clears an error", async () => {
     const { result } = renderHook(() => useAuthStore(), { wrapper });
-    act(() => result.current.authActions.authError("error1"));
+    void act(() => result.current.authActions.authError("error1"));
     expect(result.current.authState.error).toBe("error1");
-    act(() => result.current.authActions.clearError());
+    void act(() => result.current.authActions.clearError());
```

Comment:
I will remove these - I used `no-floating-promises` to find missing awaits in tests (didn't help)